packages/shared/locales/en.json

```diff
@@ -406,6 +411,12 @@
                 "title": "Error log",
                 "description": "View errors to debug an issue"
             },
+            "crashReporting": {
+                "title": "Crash reports",
+                "body": "Help the developers improve Firefly by automatically sending diagnostic data when an error or crash occurs",
+                "checkbox": "Send crash reports",
+                "advice": "Please restart the Firefly application to {sendCrashReports, select, true {start sharing crash reports} false {stop sharing crash reports} other {apply this change}}"
```

Comment:
Not sure whether this message needs to be clarified — on macOS, closing the window and re-opening Firefly (as opposed to quitting) may not apply this change:

https://github.com/iotaledger/firefly/blob/366a5d503e8671b6db46d2c9eca4314669f1aec1/packages/desktop/electron/main.js#L326-L333

Reply:
Yes that's true. I suppose it wouldn't really matter to have the text hint for this; wouldn't say users particularly care about it. Will remove!